code/renderers/react/src/public-types.ts

```diff
@@ -124,3 +127,11 @@ export type Story<TArgs = Args> = StoryFn<TArgs>;
  * ```
  */
 export type ComponentStory<T extends JSXElement> = ComponentStoryFn<T>;
+
+/**
+ * @deprecated Use Decorator instead.
+ */
+export type DecoratorFn = DecoratorFunction<ReactRenderer>;
+export type Decorator<TArgs = StrictArgs> = DecoratorFunction<ReactRenderer, TArgs>;
+export type Loader<TArgs = StrictArgs> = LoaderFunction<ReactRenderer, TArgs>;
+export type StoryContext<TArgs = StrictArgs> = GenericStoryContext<ReactRenderer, TArgs>;
```

Comment:
I fee like we are still a bit inconsistent in how we name things.

The generic version of a `Decorator` is a `DecoratorFunction`, but the generic version of a `StoryContext` is a `StoryContext`.

Should we try and nail this down?

Reply:
I think it would be good to also rename `DecoratorFunction` to `Decorator` in `@storybook/csf`.

Reply:
Made a new ticket for this